### DISCUSSION TITLE
UI Desgin Links

### DISCUSSION THREAD

**Discussion by anomatomato** [AuDojo/AuDojo#103] [UI Design]
Short 7 min introduction: https://www.youtube.com/watch?v=wIuVvCuiJhU

https://www.uxpin.com/studio/blog/good-ui-design-characteristics/

**Comment by trang-ngn** [comment]
https://youtu.be/qyomWr_C_jA?si=MWSZ4AdY2zsz9vcj
(The Easy Way to Design Top Tier Websites)

https://youtu.be/UTHgr6NLeEw
(Make Awesome SVG Animations with CSS // 7 Useful Techniques)
